### PR TITLE
[TAS-1956] ♻️ Write back actual stripe fee into db

### PR DESCRIPTION
**Pull request by williamchong**

### CODE REVIEW

File: src/util/api/likernft/book/cart.ts [hunk 1]
```diff
@@ -292,6 +292,48 @@ export async function processNFTBookCartPurchase({
   return infos;
 }
 
+async function updateNFTBooCartPostCheckoutFeeInfo({
```
Comment (nwingt) [inline src/util/api/likernft/book/cart.ts]: updateNFTBoo`k`CartPostCheckoutFeeInfo

File: src/util/api/likernft/book/purchase.ts [hunk 2]
```diff
@@ -1077,25 +1077,51 @@ export async function sendNFTBookPurchaseEmail({
...
+      'likerLandFeeAmount',
+      'likerLandCommission',
+      'channelCommission',
+      'likerLandArtFee',
+      'customPriceDiff',
+    ].forEach((key) => {
+      if (typeof newFeeInfo[key] === 'number') {
+        newFeeInfo[key] = Math.round(newFeeInfo[key] * discountRate);
+      }
+    });
```
Comment (nwingt) [inline src/util/api/likernft/book/purchase.ts]: This logic keeps repeating, can be refactorized?

**Comment by williamchong** [comment]
@nwingt I have removed 1 commit from this PR for safer deploy to prod
Please review again if this one commit can be deployed first

File: src/util/api/likernft/book/cart.ts (hunk 1 above)
Comment (williamchong) [reply src/util/api/likernft/book/cart.ts]: Will fix in #737

File: src/util/api/likernft/book/purchase.ts (hunk 2 above)
Comment (williamchong) [reply src/util/api/likernft/book/purchase.ts]: Will fix in #737

**Review by nwingt — APPROVED** [review]

**Review by nwingt — APPROVED** [review]

**Pull request merged**